### PR TITLE
CI-CD Pipeline for Provisioner team with Granular images deployment.

### PR DESCRIPTION
**Pull request by ashukakkar**
# Problem Statement
- Pipeline with Granular images deployment.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: devops/ci/update_solution.sh [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+function install_yq() {
+    pip3 show yq && pip3 uninstall yq -y
+    wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/${YQ_BINARY}.tar.gz -O - | tar xz && mv ${YQ_BINARY} /usr/bin/yq
+    if [ -f /usr/local/bin/yq ]; then rm -rf /usr/local/bin/yq; fi
+    ln -s /usr/bin/yq /usr/local/bin/yq
+}
+
+function update_solution_config() {
+    pushd $WORKSPACE/devops/ci
```
Comment (tanujashinde0405) [inline devops/ci/update_solution.sh:33]: add double-quotes to "$WORKSPACE"

Comment (tanujashinde0405) [reply devops/ci/update_solution.sh:33]: add double quotes to all variables in this script

**Review by tanujashinde0405 — CHANGES_REQUESTED** [review]

**Comment by ashukakkar** [comment]
As discussed the Jenkins code will go to RE repo. So, closing this PR.
Raised another PR for RE repo: https://github.com/Seagate/cortx-re/pull/935

**Pull request closed without merging**